src/material-examples/autocomplete-display/autocomplete-display-example.ts

```diff
@@ -30,9 +30,11 @@ export class AutocompleteDisplayExample {
 
   ngOnInit() {
     this.filteredOptions = this.myControl.valueChanges
-        .startWith(null)
-        .map(user => user && typeof user === 'object' ? user.name : user)
-        .map(name => name ? this.filter(name) : this.options.slice());
+      .pipe(
+        startWith({} as User),
```

Comment:
Is there any reason to change from `null` to `{} as User`?

Reply:
Having it stay as `null` breaks `strictNullChecks`.